cmd/task/task.go

```diff
@@ -81,6 +82,7 @@ func main() {
 	pflag.StringVarP(&dir, "dir", "d", "", "sets directory of execution")
 	pflag.StringVarP(&entrypoint, "taskfile", "t", "", `choose which Taskfile to run. Defaults to "Taskfile.yml"`)
 	pflag.StringVarP(&output, "output", "o", "", "sets output style: [interleaved|group|prefixed]")
+	pflag.IntVarP(&jobs, "jobs", "j", 0, "limit number of jobs (tasks) to run simultaneously")
```

Comment:
What about calling this "threads"? 🤔 

Probably not the best suggestion, but I think that "jobs" is too generic and it may not be clear to the user we're talking about concurrency here.

Let me know if you have more suggestions.

Reply:
`--concurrency` feels good despite being a bit long (perhaps we could use `-C` as an alias).

Reply:
Yea I'm alright with that! I was mostly following the convention of `make` I wasn't sure what the best name was.

Reply:
Everything has been updated when you have time to look again.

Reply:
Hey @RossHammer,

Just commenting since it has been some time...

I have a few concerns and I think I'll postpone to be re-reviewed after the v3 release (which is more than late already).

Sorry, been a but busy recently...